### ISSUE TITLE
Access rights for global settings

### ISSUE THREAD

**Issue by gooutbefree** [inventree/InvenTree#3730]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### Describe the bug*

I tried to limit the access rights in a way that the user can not change the global settings in the settings dialog (below the user settings). I think only an admin user should be able to change these settings.
I can't find any settings to limit access to global settings dialog

### Steps to Reproduce

- set-up a user without admin rights
- Go in the webui to Setting -> global settings
-

### Expected behavior

Only a user with admin rights can change global configuration setting in the settings dialog

### Deployment Method

- [ ] Docker
- [X] Bare metal

### Version Information

# Version Information:
InvenTree-Version: 0.8.3
Django Version: 3.2.15
Commit Hash: f9d2b149c
Commit Date: 2022-09-23
Database: mysql
Debug-Mode: False
Deployed using Docker: False

### Relevant log output

```shell
no log
```

**Comment by SchrodingersGat** [comment]
@gooutbefree this should already be the case. I believe that the global settings can only be accessed by users with "staff" permission. But you are saying that any user can view the global settings?

For example on the demo server, if you log in with the "reader" account you only see user settings:

![image](https://user-images.githubusercontent.com/10080325/192901552-42bfb72a-7257-430d-a297-cfd49ca1e035.png)

**Comment by gooutbefree** [comment]
@SchrodingersGat You are right. It's working fine as long as staff user is a synonymous for administrator ;-)
I added LDAP authentication in the settings.py file to our ldap server. Other applications use staff group for standard users and only superusers for admin. I'm sorry for the trouble, I haven't considers the staff flag. I disabled staff on a normal user but with the next login the flag was enabled again.
